src/main/java/api/ApiRequestBuilder.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package api;
+
+import java.util.Map;
+
+/**
+ * Helps to build an ApiRequest entity.
+ */
+public class ApiRequestBuilder implements IBuilder {
```

Comment:
The ApiRequest being a wrapper and making a builder out of the ApiRequest may add needed complexity, the required operations could probably be handled just in the ApiRequest.

Reply:
Already fixed